addons/website/views/snippets/snippets.xml

```diff
@@ -552,7 +552,7 @@
          data-no-check="true"/>
 
     <div data-js="WebsiteLevelColor"
-         data-selector="#wrapwrap > header"
+         data-selector="body:not(.oe_editor_translation) #wrapwrap > header"
```

Comment:
You're right, I missed that part. As I saw the option targeting `"#wrapwrap > header"` being opened at some point, I thought some snippet could still be opened in translation. But the only reason it opened is because of the `_toggleMegaMenu` that called `activateSnippet`.